yb-voyager/cmd/exportData.go

```diff
@@ -338,6 +338,10 @@ func exportData() bool {
 	fmt.Printf("num tables to export: %d\n", len(tableListToDisplay))
 	utils.PrintAndLog("table list for data export: %v", tableListToDisplay)
 
+	if source.DBType == POSTGRESQL {
+		utils.PrintAndLog("Only the sequences that are attached to the above tables will be restored during the migration.")
+	}
```

Comment:
How about also mentioning something like this - `tables being exported(refer above list) will be ...`

Reason being, lets say user didn't specify any table list, and then they don't necessarily need to go through the printed table list to ensure. They can infer that sequences attached to all the tables(being exported) will be restored.